scripts/debug_tools/prepare_all_cmd.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+
+
+def execute(cmd):
+    print("Executing command: " + ' '.join(cmd))
+    try:
+        proc = subprocess.Popen(cmd,
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                )
+        out, err = proc.communicate()
+
+        print("stdout:\n\n" + out.decode("utf-8"))
+        print("stderr:\n\n" + err.decode("utf-8"))
+
+        if proc.returncode != 0:
+            print('Unsuccessful run: "' + ' '.join(cmd) + '"')
+            raise Exception("Unsuccessful run of command.")
+        return out
+    except OSError:
+        print('Failed to run: "' + ' '.join(cmd) + '"')
+        raise
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Prepare all commands '
+        'to execute in local environmennt for debugging.')
+    parser.add_argument('--clang', required=True)
+    parser.add_argument('--libericsson', required=True)
```

Comment:
Should we expose ericsson in APIs? Maybe we should call this clangplugin? Because that library is actually a clang plugin. Also, I would make this optional, so open source users who do not use any clang plugin can also benefit from this.

Reply:
Removed `--libericsson`, added `--clang_plugin_name` and `--clang_plugin_path`